src/eval-frame/actions/actions.js

```diff
@@ -161,7 +161,7 @@ function evaluateCode(code, language, state, evalId) {
       );
     };
 
-    return ensureLanguageAvailable(language, state, dispatch)
+    return ensureLanguageAvailable(language, state, evalId, dispatch)
```

Comment:
as below, might be missing it, but it looks like evalId is not yet used in this function?